docs/index.html

```diff
@@ -6,12 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
+    <title>CRC1451 Catalog</title>
+    <meta name="description" content="The CRC1451 data catalog for the German Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```

Comment:
```suggestion
    <meta name="description" content="The CRC1451 data catalog for the Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```
Probably doesn't need to say German - we are funded by DFG but we have international collaborations as well.